### ISSUE TITLE
Add Outcome Related to Storylines

### ISSUE THREAD

**Issue by SuzanneKTaylor** [w3c/wcag3#89]
The [Introduction to the May 2024 draft of WCAG 3.0](https://www.w3.org/blog/2024/the-wcag-3-working-draft-update-is-ready-for-your-review/) asked  "What outcomes needed to make web content accessible are missing?"

While Disturbing Content and Triggers are included, often content interweaves multiple storylines to keep a wider audience engaged. Just hiding specific items in a storyline may create an untidy experience, while also not fully protecting the user from whatever they find triggering, since the context would remain.  In addition, users might have specific interests and be unable to focus through other storylines. Also, following interwoven storylines requires cognitive skills such as remembering and distinguishing between plots, characters, locations, and times. Some users may find it easier to experience one storyline at a time, even if they plan to view all of the storylines. An outcome about being able to select storylines could help:

Control of Storylines
Individual storylines are programmatically distinguishable, or users can choose storylines.

**Comment by GreggVan** [comment]
Here is a place where I  think AI based AT is needed.    We can't write content that does not have a trigger for anyone.  Depending on personal experience - mentioning a grove of trees could be a trigger for someone. And I  can see us descending into a never-ending discussion of what should be considered a trigger or not.     I think this might be better handled with an AI filter that detects content that would be a trigger for someone and redacts it or whatever the individual feels is appropriate.

**Comment by SuzanneKTaylor** [comment]
I think this is true for "Disturbing Content" and for "Triggers," and I also think that even with an AI based AT subtle and complex triggers will be difficult to address, because 1) Asking a user to set up such a profile could be more triggering than the content itself and 2) Such a profile could be used for great harm if the information security is not at the right level - imagine a politician whose debate opponents can get a copy of the profile.

On the other hand, simple and common triggers could easily be addressed, such as "Don't show spiders in this video game". Those outcomes may have to stay at this sort of basic level until we know more.

The outcome I'm suggesting here is just to be able to choose storylines, when, for example, 3 are interwoven - This would  allow some complex subtle triggers to be addressed in a simple straightforward manner. So if a video shows the stories of 3 people, but you see 10 minutes on the first, then 10 minutes on the second, then 10 minutes on the third, then 5 minutes on the first, then 5 minutes on the second, then 5 minutes on the third, etc, you could choose to just watch (or not watch) any one of the full stories.
